apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTestRunner.h

```diff
@@ -23,33 +23,90 @@
 #include <memory>
 #include "AudioWorkloadTest.h"
 
+/**
+ * @class AudioWorkloadTestRunner
+ * @brief Manages the execution and results of an AudioWorkloadTest.
+ *
+ * This class encapsulates an AudioWorkloadTest instance, controls its lifecycle (start, stop),
+ * monitors its progress, and determines a pass/fail result based on criteria like XRuns.
+ *
+ * Example usage:
+ * start();
+ * while(!stopIfDone()) { getStatus(); sleep(); }
+ * getResultText();
+ *
+ */
 class AudioWorkloadTestRunner {
 public:
-    AudioWorkloadTestRunner();
+    /**
+     * @brief Default constructor.
+     */
+    AudioWorkloadTestRunner() = default;
+
+    /**
+     * @brief Destructor. Ensures the test is stopped when the runner is destroyed.
+     */
     ~AudioWorkloadTestRunner();
 
+    /**
+     * @brief Starts the audio workload test with the specified parameters.
+     * @param targetDurationMs The desired duration of the test in milliseconds.
+     * @param bufferSizeInBursts The desired buffer size in terms of multiples of framesPerBurst.
+     * @param numVoices The primary number of synthesizer voices to simulate.
+     * @param highNumVoices An alternative number of voices for alternating workload.
```

Comment:
I f I do not want to alternate, should I set this to zero or the same as numVoices?